resources/tests/build_kernel.sh

```diff
@@ -90,7 +90,9 @@ build_kernel() {
     cp $binary_path $binary_name
 
     popd >/dev/null
-    echo "Kernel binary placed in build/kernel/$binary_name"
+
+    local full_binary_name="$KERNEL_DEST_DIR/$binary_name"
+    echo "Kernel binary placed in ${full_binary_name##$FC_ROOT_DIR/}"
```

Comment:
How about preserving the original behavior?
```
diff --git a/resources/tests/build_kernel.sh b/resources/tests/build_kernel.sh
index 42b3deac..2c592a90 100755
--- a/resources/tests/build_kernel.sh
+++ b/resources/tests/build_kernel.sh
@@ -86,10 +86,11 @@ build_kernel() {
 
     make olddefconfig
     make -j $NPROC $target
-    local binary_name="vmlinux-$KERNEL_VERSION-$arch.bin"
-    cp $binary_path $binary_name
 
     popd >/dev/null
+
+    local binary_name="vmlinux-$KERNEL_VERSION-$arch.bin"
+    cp "$KERNEL_DEST_DIR/$binary_path" "$FC_KERNEL_DIR/$binary_name"
     echo "Kernel binary placed in build/kernel/$binary_name"
 }
```

Reply:
I'm really sorry. I was wrong. The original behavior was to save the kernel under `build/kernel/linux-<version>/`.
https://github.com/firecracker-microvm/firecracker/commit/fb0f950eb2ff72d5bacb1f73c6aadabd733d6a56

Your initial change preserves the original behavior.